kubernetes/istio.go

```diff
@@ -507,8 +507,8 @@ func ParseRegistryConfig(config map[string][]byte) (*RegistryConfiguration, erro
 							registry.EnvoyFilters = append(registry.EnvoyFilters, ef)
 						case "Gateway":
 							// It needs to figure out Gateway object type by apiVersion, whether it is Gateway API of Istio Gateway
-							if mItem["apiVersion"] == K8sApiNetworkingVersionV1Beta1 || mItem["apiVersion"] == K8sApiNetworkingVersionV1Alpha2 {
```

Comment:
I think we should leave it as backward compatible, to list existing v1alpha2 Gateways.

Reply:
done